### PR TITLE
Camilo/nefi 79/conversation thread paginated v1

### PR DESCRIPTION
**Pull request by ctabares-uruit**
# Description

Add the functionality to load messages and display messages in the UI with the correct format.

# Main Changes

- add `LoadChatMessagesGeneratorHook` to load messages
- add messages to `ConversationChatProvider` and `useConversationChat`
- load messages when `selectedConversation` changes in `ConversationChatProvider`
- add required props to `IncomingMessages` and `OutcomingMessages`

# Preview

### Incoming and outcoming messages

imagen

### Incoming messages only

imagen

### CODE REVIEW

File: client/src/components/Messages/IncomingMessage/index.js [hunk 1]
```diff
@@ -19,8 +20,8 @@ const IncomingMessage = () => {
 
   return (
     <BubbleChatMesage
-      header="Via Gmail - Yesterday"
-      title="Claim auto"
+      header={`Via ${channel} - ${getPastTimeLabel(received_at)}`}
+      title={from}
```
Comment (alphonse92) [inline client/src/components/Messages/IncomingMessage/index.js:24]: Title should be the conversation display_name

File: client/src/components/Messages/IncomingMessage/index.js [hunk 2]
```diff
@@ -6,8 +6,9 @@ import Vertical from '../../layout/Vertical';
 import Horizontal from '../../layout/Horizontal';
 import Text from '../../custom/typography/Text';
 import BubbleChatMesage from '../BubbleChatMessage';
+import {getPastTimeLabel} from '../../../utils/dates';
 
-const IncomingMessage = () => {
+const IncomingMessage = ({from, channel, received_at, preview_message}) => {
```
Comment (alphonse92) [inline client/src/components/Messages/IncomingMessage/index.js:11]: isn't preview message, is just message, replace all the places you wrote "preview_message" in this component. The preview message is just temporal

File: client/src/components/Messages/OutcomingMessage/index.js [hunk 3]
```diff
@@ -7,16 +7,14 @@ import Horizontal from '../../layout/Horizontal';
 import Text from '../../custom/typography/Text';
 import BubbleChatMesage from '../BubbleChatMessage';
 
-const OutgoingMessage = () => {
+const OutgoingMessage = ({preview_message}) => {
```
Comment (alphonse92) [inline client/src/components/Messages/OutcomingMessage/index.js:10]: same here, isn't preview message

File: client/src/components/Messages/OutcomingMessage/index.js [hunk 4]
```diff
@@ -7,16 +7,14 @@ import Horizontal from '../../layout/Horizontal';
...
   const {variables} = useTheme();
   const {bubbleMyMessageBackground, bubbleMyMessageColor} = variables;
 
   return (
     <BubbleChatMesage toRight backgroundColor={bubbleMyMessageBackground}>
       <Text sx={{color: bubbleMyMessageColor, textAlign: 'left'}}>
-        Hi Patty! It is a pleasure to communicate with you. I wanted to let you
-        know that I have not received answers about the claim that I entered on
-        the web, and since I spoke with you last I once imagined...
+        {preview_message}
```
Comment (alphonse92) [inline client/src/components/Messages/OutcomingMessage/index.js:17]: just message

File: client/src/hooks/conversations/chat/LoadChatMessagesGeneratorHook.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import {API_URL} from '../../../configs/config';
```
Comment (alphonse92) [inline client/src/hooks/conversations/chat/LoadChatMessagesGeneratorHook.js:1]: Just to confirm: this implementation will be replaced in the second PR, right?

**Review by alphonse92 — CHANGES_REQUESTED** [review]

File: client/src/hooks/conversations/chat/LoadChatMessagesGeneratorHook.js (hunk 5 above)
Comment (ctabares-uruit) [reply client/src/hooks/conversations/chat/LoadChatMessagesGeneratorHook.js:1]: Yes, we are going to use `RequestGeneratorHook` for v2 with pagination.

File: client/src/components/Messages/IncomingMessage/index.js (hunk 1 above)
Comment (ctabares-uruit) [reply client/src/components/Messages/IncomingMessage/index.js]: done

File: client/src/components/Messages/IncomingMessage/index.js (hunk 2 above)
Comment (ctabares-uruit) [reply client/src/components/Messages/IncomingMessage/index.js]: done

File: client/src/components/Messages/OutcomingMessage/index.js (hunk 3 above)
Comment (ctabares-uruit) [reply client/src/components/Messages/OutcomingMessage/index.js]: done

File: client/src/components/Messages/OutcomingMessage/index.js (hunk 4 above)
Comment (ctabares-uruit) [reply client/src/components/Messages/OutcomingMessage/index.js]: done

**Review by alphonse92 — APPROVED** [review]
Looks great!

**Pull request merged**